docs/dev/guidelines/client-tests.rst

```diff
@@ -0,0 +1,257 @@
+******
+Client Tests
+******
+
+**If you are new to client testing, it is highly recommended that you work through the testing part of the angular tutorial:** https://angular.io/guide/testing
```

Comment:
This is a bit confusing, as it ends with **:** 
One could think that the testing part of the angular tutorial is what is following.
I suggested to link the testing part of the angular tutorial as link of the words **testing part** below. But besides, I would end that sentence with **.** and maybe start a new sentence that outlines that the following describes the best practices in the Artemis context or something similar.

Reply:
Applied the suggestions as defined in another comments by you.